custom/conf/app.example.ini

```diff
@@ -1143,6 +1143,83 @@ LEVEL = Info
 ;PROJECT_BOARD_BASIC_KANBAN_TYPE = To Do, In Progress, Done
 ;PROJECT_BOARD_BUG_TRIAGE_TYPE = Needs Triage, High Priority, Low Priority, Closed
 
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;[project.automation]
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;ENABLED = false
+;MAX_RULES_PER_PROJECT = 25
+
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;[project.automation.kanban]
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;;
+;; Default rules when creating an automated Kanban project.
+;; Each rule can specify a list of key/value pairs:
+;; - trigger (required)  see commands below
+;; - action (required)   see commands below
+;; - target (optional)   the rule is only active for these issue types
+;;                       one of: issue, pr, default
+;;                       can be comma-separated list (e.g. issue,pr)
+;;                       the 'default' depends on the action, but most often
+;;                       it is the same as issue,pr (notable exceptions are
+;;                       commands 'xref', 'approve' and 'assign_reviewer')
+;;                       (default value is target=default)
+;; - context (optional)  the rule is only active in this context
+;;                       one of: column:{column}, project
+;;                       where {column} is a column title
+;;                       (default value is context=project)
+;;
+;; Commands can have arguments using the command:argument syntax.
+;; They are available both as a trigger and as an action.
+;; Available commands:
+;; - move:{column}   where {column} is a column title
+;; - status:{status} where {status} is one of: closed, reopened
+;; - assign_project  assigns the current project to the issue/pr
+;; - assign_reviewer adds the current user as a requested reviewer
+;; - assign          assigns the current user to the issue/pr
+;;                   when used as a trigger, it triggers when issue/pr is no longer unassigned
+;; - unassign        unassign the current user from the issue/pr
+;;                   when used as a trigger, it triggers when issue/pr becomes unassigned
+;; - approve         approves a pull request (default target=pr)
+;; - unapprove       unapproves a pull request (default target=pr)
+;; - xref:{action}   a cross-reference is made between issue and pull request
+;;                   where {action} is one of: none, closes, reopens, neutered
+;;                   When xref is used as a trigger, target=pr can target a
+;;                   pull request which will close an issue in the current project,
+;;                   even if that pull request is not currently assigned to the project.
+;;                   (default target=issue)
```

Comment:
Hmm… This does not sound like a good idea.
It seems like we have different ideas what `xref`s are.
An `xref` is for me an `external reference`, so i.e. I mention PR `#1` in issue `#2`.
How does `opens`/`closes`/… fit into this?

Reply:
Would renaming this to `pr:closes` or something like that (and only keeping `closes` as an action) be better?  Because as I said I think that is the only use-case which might be interesting right now.